codes/quantum/qubits/quantum_cyclic.yml

```diff
@@ -0,0 +1,117 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+# Use UTF-8 unicode encoding
+# AMS-TeX commands are rendered inside \( ... \) using MathJaX.
+# Allowed external bibliographic references are
+#    \cite{arXiv:#.#} or \cite{arXiv:quant-ph/#},
+#    \cite{doi:#}, or, as a last resort
+#    \cite{manual:{(enter citation line incl. author and year here)}}
+# External websites such as code tables, coding theory packages, github pages linked as
+#    \url{https://example.com/example}
+#    \href{https://example.com/example}{link text}
+# Internal references to codes are 
+#    \hyperref[code:code_id]{link text}
+# Delete instructional comments when submitting
+
+# code id, physical, logical are all lower case
+code_id: quantum_cyclic
+physical: qubits
+logical: qubits
+
+# Apostrophes are denoted by two apostrophe characters, i.e., ''
+# Code title is singular
+name: 'Quantum-Cyclic code'
+
+# First reference with the code (optional).
+introduced: '\cite{arXiv:1007.1697}'
+
+# Anything applicable to a larger parent set of codes (see below) should go in
+# that entry instead of here.
+description: 'Every linear stabilizer code \(C\) has an associated totally isotropic subspace \(S\subset F_2^{2n}\). Quantum cyclic codes are those codes where \(S\) and its centralizer are ideals in the polynomial ring \(F_4[X]/X^n-1\).'
```

Comment:
What do you mean by "has"; is there a more precise way to identify this subspace with the stabilizer info?

Reply:
I was too deep into the paper and forgot the simple description of quantum cyclic codes. I've updated with the simple definition.